mod_reforged/hooks/states/world_state.nut

```diff
@@ -13,4 +13,30 @@
 
     	return __original(_properties, _isPlayerInitiated, _isCombatantsVisible, _allowFormationPicking);
     }
+
+	q.onMouseInput = @(__original) function( _mouse )
+	{
+		local ret = __original(_mouse);
+		if (ret == false)	// If the original function wasn't able to process the mouseclick we try to do that with increased interaction range
+		{
+			local isEscorting = this.m.EscortedEntity != null && !this.m.EscortedEntity.isNull();
+			if (isEscorting && _mouse.getState() == 1 && !this.isInCameraMovementMode() && !this.m.WasInCameraMovementMode)
+			{
+				foreach (entity in ::World.getAllEntitiesAndOneLocationAtPos(::World.getCamera().screenToWorld(_mouse.getX(), _mouse.getY()), 1.0))
+				{
+					if (entity.isParty()) continue;
+					if (!entity.isEnterable()) continue;
+					if (!entity.isAlliedWithPlayer()) continue;
```

Comment:
I really think the following looks and reads much better:
```squirrel
if (entity.isParty() || !entity.isEnterable() || !entity.isAlliedWithPlayer())
   continue;
```
than these 3 split very small checks. But it's up to you.